image/templates/helm/stackrox-secured-cluster/internal/defaults/30-base-config.yaml.htpl

```diff
@@ -114,6 +114,10 @@ collector:
     enabled: {{ ne ._rox.env.openshift 0 }}
   runtimeConfig:
     enabled: true
+    networking:
+      externalIps:
+        enable: false
+      per_container_rate_limit: 1024
```

Comment:
This seems wrong (snake_case vs camelCase).

Reply:
Yes. Collector initially expected `per_container_rate_limit` but that was changed to `perContainerRateLimit`, but that was corrected. I forgot to correct it here, when I corrected it elsewhere.

Reply:
Yes. Collector initially expected `per_container_rate_limit` but that was changed to `perContainerRateLimit`, but that was corrected. I forgot to correct it here, when I corrected it elsewhere.